src/ray/gcs/gcs_server/gcs_autoscaler_state_manager.cc

```diff
@@ -131,7 +128,8 @@ void GcsAutoscalerStateManager::GetPendingGangResourceRequests(
   }
 
   // Iterate through each placement group load.
-  for (const auto &pg_data : placement_group_load->placement_group_data()) {
+  for (auto &&pg_data :
```

Comment:
I rarely see people doing this... curious why not 
```
for (auto& pg_data: *mutable) {
  // do whatever move if you like
}
```

Reply:
it makes it work for "use after move" linting, basically calling move on it will mark it as "used" and then if we any access to it will be flagged. Doesn't functionally do anything

Reply:
Gotcha!